### PR TITLE
Require configuration ids

### PR DESCRIPTION
**Pull request by zdrazil**
#### Check during review

- [ ] JSON example extended.
  - [ ] New properties are added to the correct place in the JSON.
- [ ] New properties in the table are added to the correct place.
- [ ] Correct formatting:
  - [ ] Spacing is consistent between titles, sections, tables, ...
  - [ ] Correct JSON format - indentation.
- [ ] DateTime properties should always be defined in ISO format.

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Changelog
+
+## 19.11.2020
+
+* Stopped supporting hotel ids in URL.
```
Comment (koprivajakub) [inline changelog.md:5]:
```suggestion
* Stopped support for hotel/enterprise ids in URL and in the configuration of a javascript client
```

File: distributor-api-v1/changelog.md [hunk 2]
```diff
@@ -1,8 +1,13 @@
 # Changelog
 
+## 19.11.2020
+
+* Made `ConfigurationId` parameter in [CreateReservationGroup](operations.md#create-reservation-group) endpoint required.
+* Added `ConfigurationId` parameter to [GetAvailability](operations.md#get-availability) endpoint.
```
Comment (koprivajakub) [inline distributor-api-v1/changelog.md:6]: I would add a note here as well that the previous variant of sending `HotelId` is deprecated

File: changelog.md (hunk 1 above)
Comment (koprivajakub) [reply changelog.md:5]: Ah sorry, it was never mentioned. But I would say we can be better safe than sorry here.

**Review by koprivajakub — APPROVED** [review]

File: distributor-api-v1/changelog.md (hunk 2 above)
Comment (zdrazil) [reply distributor-api-v1/changelog.md:6]: Does writing it like this sound alright to you?:

>* Previous variant of sending only `HotelId` without `ConfigurationId` to these endpoints is deprecated.

 Right now we still need to send both `HotelId` and `ConfigurationId`. The api doesn't yet work with only `ConfigurationId`.
If you omit `HotelId` from the request, the api will respond with message `Invalid HotelId.`

Comment (zdrazil) [reply distributor-api-v1/changelog.md:6]: In the end I've added this: 
> Previous variant of sending `HotelId` without `ConfigurationId` to these endpoints was deprecated.

If it's not alright, please let me know and I'll reformulate it as followup.

**Pull request merged**